sigstore/_cli.py

```diff
@@ -34,10 +35,24 @@ def main():
     type=click.File("rb"),
     default=resources.open_binary("sigstore._store", "ctfe.pub"),
 )
+@click.option(
+    "oidc_client_id", "--oidc-client-id", type=click.STRING, default="sigstore"
+)
+@click.option(
+    "oidc_client_secret", "--oidc-client-secret", type=click.STRING, default=str()
+)
+@click.option(
+    "oidc_issuer",
+    "--oidc-issuer",
+    type=click.STRING,
+    default="https://oauth2.sigstore.dev/auth",
```

Comment:
Might want to give this a `metavar="URL"`, so that the CLI doesn't just render its as `--oidc-issuer TEXT`.